content/en/integrations/guide/storage-monitoring-setup.md

```diff
@@ -9,7 +9,7 @@ private: true
 
 Storage Monitoring for Amazon S3 provides deep, prefix-level analytics to help you understand exactly how your storage is being used, detect potential issues before they impact operations, and make data-driven decisions about storage optimization. Use these insights to help you track storage growth, investigate access patterns, and optimize costs.
 
-This guide explains how to configure Storage Monitoring in Datadog for your S3 buckets. You can set this up either manually or using the provided CloudFormation templates. Access your Storage Monitoring data by navigating to **Infrastructure -> Resource Catalog -> Monitoring -> S3 Buckets**.
+This guide explains how to configure Storage Monitoring in Datadog for your S3 buckets. You can set this up either manually or using the provided CloudFormation templates. Access your Storage Monitoring data by navigating to **Infrastructure -> [Resource Catalog][107] -> Monitoring -> S3 Buckets**.
```

Comment:
Is there anywhere else you can reference the Resource Catalog? I think in this section, a user might expect that this would link out to the app, rather than the documentation.

```suggestion
This guide explains how to configure Storage Monitoring in Datadog for your S3 buckets. You can set this up either manually or using the provided CloudFormation templates. Access your Storage Monitoring data by navigating to **Infrastructure -> Resource Catalog -> Monitoring -> S3 Buckets**. 

To learn more about the Resource Catalog, see the [Resource Catalog][107] documentation.
```

Reply:
Totally fair!